Makefile

```diff
@@ -1,19 +1,22 @@
 NAME=nomad-web
 
+DOCKER_MACHINE_NAME := $(shell echo $${DOCKER_MACHINE_NAME:-default})
+DOCKER_IP=$(shell docker-machine ip ${DOCKER_MACHINE_NAME})
+
 help:
+	# bootstrap - initial command that builds everything for development
 	# build - initial command that builds everything
 	# runserver - runs nomadboard
 	# django-shell - runs django's shell
 
-# start: build wakeup-database migrate collectstatic
+bootstrap: build wakeup-database migrate
```

Comment:
this doesn't work, database actually isn't woken up when you call the `migrate` command.